### DISCUSSION TITLE
Getting error While Creating a new patient via web api

### DISCUSSION THREAD

**Discussion by trainingmanorama** [dcm4che/dcm4chee-arc-light#3917] [Q&A] [answered]
Getting error While Creating a new patient via web api

{"errorMessage":"No Web Application with DCM4CHEE_ARC_AET service class found for Application Entity: DCM4CHEE"}

![image](https://user-images.githubusercontent.com/122001767/210713021-25b80e1c-86c4-4c97-a284-aef6bc772824.png)

**Comment by gunterze** [comment, marked as answer]
Check the configured _Web Service Classes_ for _Web Application_ `DCM4CHEE`, which contains _Web Service Class_ `DCM4CHEE_ARC_AET` by default:
![Screenshot from 2023-01-05 10-46-36](https://user-images.githubusercontent.com/400095/210750604-1660556a-6c34-4021-b870-4861bbf9a317.png)
![Screenshot from 2023-01-05 10-47-14](https://user-images.githubusercontent.com/400095/210750624-c873104b-73b4-492b-88c2-76d01c650973.png)
